### ISSUE TITLE
conflict when using -l and logger with watchpid

### ISSUE THREAD

**Issue by nbari** [immortal/immortal#7]
When logging to a file using -l or -logger

The watchpid line is not printed but seems to be working, the expected logs should be somethign like:

```
Mar 17 01:48:46 nbari-mini immortal[82586]: PID 82591 terminated, exit status 0 [314.523ms user  72.499ms sys  37.000382188s up]
Mar 17 01:48:46 nbari-mini immortal[82586]: Watching pid 83737 on file: ./unicorn.pid
```

Also, the process remains up "green", but can't send signals to it

**Comment by nbari** [comment]
possible related https://github.com/golang/go/issues/10338

**Comment by nbari** [comment]
Fixed in 0.10.1
